extensions/amp-story/1.0/bookend/components/article.js

```diff
@@ -60,14 +68,18 @@ export class ArticleComponent {
    * */
   build(articleJson) {
     const article = {
-      type: articleJson['type'],
-      title: articleJson['title'],
-      url: articleJson['url'],
-      domainName: parseUrlDeprecated(articleJson['url']).hostname,
+      type: articleJson.type,
+      title: articleJson.title,
+      url: articleJson.url,
+      domainName: parseUrlDeprecated(articleJson.url).hostname,
     };
 
-    if (articleJson['image']) {
-      article.image = articleJson['image'];
+    if (articleJson.image) {
+      article.image = articleJson.image;
+    }
+
+    if (articleJson.ampdoc) {
+      article.ampdoc = articleJson.ampdoc;
```

Comment:
This is not a problem yet, but I'd recommend using brackets when dereferencing JSON. Context: https://developers.google.com/closure/compiler/docs/api-tutorial3#propnames

Reply:
Interesting, thanks for the link!